### DISCUSSION TITLE
CommunityToolkit.Maui MAUI SR1 Compatability

### DISCUSSION THREAD

**Discussion by Orgbrat** [CommunityToolkit/Maui#1672] [Q&A] [answered]
Is the CommunityToolkit.Maui 7.0.1 compatible with the .NET MAUI SR1 release? If Not when will we get a compatible version?

**Comment by VladislavAntonyuk** [comment, marked as answer]
It is compatible. At least I don’t know any issues related to it.

**Comment by CliffAgius** [comment]
Closed as answered.
